src/Aspire.ProjectTemplates/templates/aspire-starter/AspireStarterApplication.1.ServiceDefaults/AspireStarterApplication.1.ServiceDefaults.csproj

```diff
@@ -1,25 +1,22 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <OutputType>Library</OutputType>
     <TargetFramework>net8.0</TargetFramework>
     <ImplicitUsings>enable</ImplicitUsings>
     <Nullable>enable</Nullable>
     <IsAspireSharedProject>true</IsAspireSharedProject>
   </PropertyGroup>
 
   <ItemGroup>
-    <FrameworkReference Include="Microsoft.AspNetCore.App" />
```

Comment:
The reason this "works" to take out this FrameworkReference is because `OpenTelemetry.Instrumentation.AspNetCore` also has a FrameworkReference to AspNetCore.App.

I'd prefer we keep this `<FrameworkReference Include="Microsoft.AspNetCore.App" />` in the ServiceDefaults csproj and not rely on having the OTel package bring it in for us. It makes the dependences of this library more explict. If someone would remove the OTel.AspNetCore code from the library, and removed the PackageReference, the library wouldn't be able to be built anymore.